doc/release-notes/6.8-release-notes.md

```diff
@@ -0,0 +1,343 @@
+
+```shell
+$PAYARA/bin/asadmin undeploy dataverse-6.7.1
+```
+
+3\. Download and deploy this version
+
+```shell
+wget https://github.com/IQSS/dataverse/releases/download/v6.8/dataverse-6.8.war
+$PAYARA/bin/asadmin deploy dataverse-6.8.war
+```
+
+Note: if you have any trouble deploying, stop Payara, remove the following directories, start Payara, and try to deploy again.
+
+```shell
+sudo service payara stop
+sudo rm -rf $PAYARA/glassfish/domains/domain1/generated
+sudo rm -rf $PAYARA/glassfish/domains/domain1/osgi-cache
+sudo rm -rf $PAYARA/glassfish/domains/domain1/lib/databases
+sudo service payara start
+```
+
+4\. Update metadata blocks
+
+These changes reflect incremental improvements made to the handling of core metadata fields.
+
+Reload the citation.tsv file to handle the commons-lang3 change mentioned above.
+
+Expect the loading of the citation block to take several seconds because of its size (especially due to the number of languages).
+
+```shell
+wget https://raw.githubusercontent.com/IQSS/dataverse/v6.8/scripts/api/data/metadatablocks/citation.tsv
+
+curl http://localhost:8080/api/admin/datasetfield/load -H "Content-type: text/tab-separated-values" -X POST --upload-file citation.tsv
+```
+
+5\. For installations with internationalization or text customizations:
+
+Please remember to update translations via [Dataverse language packs](https://github.com/GlobalDataverseCommunityConsortium/dataverse-language-packs).
+
+If you have text customizations you can get the latest English files from <https://github.com/IQSS/dataverse/tree/v6.8/src/main/java/propertyFiles>.
+
+6\. Update Solr schema and reindex
+
+Due to changes in the Solr schema (the addition of field "datasetCount"), updating the Solr schema and reindexing is required.
+
+Download the updated `schema.xml` file:
+
+```shell
+wget https://raw.githubusercontent.com/IQSS/dataverse/v6.8/conf/solr/schema.xml
+cp schema.xml /usr/local/solr/solr-9.8.0/server/solr/collection1/conf
+```
+
+6a\. For installations with additional metadata blocks or external controlled vocabulary scripts, update fields
+
+- Stop Solr instance (usually `service solr stop`, depending on Solr installation/OS, see the [Installation Guide](https://guides.dataverse.org/en/6.8/installation/prerequisites.html#solr-init-script)).
+
+- Run the `update-fields.sh` script that we supply, as in the example below (modify the command lines as needed to reflect the correct path of your Solr installation):
+
+```shell
+wget https://raw.githubusercontent.com/IQSS/dataverse/v6.8/conf/solr/update-fields.sh
+chmod +x update-fields.sh
+curl "http://localhost:8080/api/admin/index/solr/schema" | ./update-fields.sh /usr/local/solr/solr-9.8.0/server/solr/collection1/conf/schema.xml
+```
+
+Note that Docker-based installations use a different directory: `solr/data/data/collection1/conf/schema.xml`.
+
+- Start Solr instance (usually `service solr start` depending on Solr/OS).
+
+7\. Reindex Solr
```

Comment:
Is the new `datasetCount` field the only reason we are suggesting a reindex?

Reply:
I'm still debating if it's worth mentioning. It's kind of criminal to tell someone with a database the size of ours to reindex everything (a monumental task) when it's only the collections that need to be reindexed. But then there are few if any instances the size of ours. 

But I feel like it could be a good practice to always tell people why a reindex is on the checklist. Just so that an instance can decide if that's something they need.